model/service/rdf/RdfBatteryClassService.php

```diff
@@ -50,23 +49,18 @@ public function getRootClass()
     public function deleteResource(\core_kernel_classes_Resource $resource)
     {
         try {
-            $this->getEventManager()->trigger(new BatteryBeforeRemoveEvent($resource->getUri()));
             $result = $resource->delete();
+            $this->getEventManager()->trigger(new BatteryRemovedEvent($resource, []));
         }
         catch (\Throwable $e) {
             // If some BatteryBeforeRemoveEvent event handler wants to prevent battery removing and
```

Comment:
Please delete the comment, it's not relevant anymore, because it's no longer possible to cancel battery removal in a `BatteryBeforeRemoveEvent` listener.

Reply:
So maybe I should remove this try catch block? No?

Reply:
Why we deleted BatteryBeforeRemoveEvent triggering? It's used in taoNccerDelivery TAO-7230_assignments_rework branch

Reply:
oops :)

Reply:
It's really difficult to work with two different branches here. @siarhei-baradzin where is this called in this TAO-7230_assignments_rework branch?